index.html

```diff
@@ -56,6 +56,148 @@ <h2>I'm Cristhian. Glad to see you!</h2>
+              <div class="technologys">
+                <ul class="list-technologys">
+                  <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+                  <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+                  <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
+                </ul>
+              </div>
+              <button type="button">See project</button>
+            </div>
+          </div>
+          <div class="card">
+            <img  class="img-project" src="/img/Snapshoot-Portfolio4.svg" alt="">
+            <div class="description-project multi">
+              <h3>Multi-Post Stories</h3>
+              <div class="frame">
+                <p>CANOPY</p>
+                <img src="/img/Counter.svg">
+                <p>Back End Dev</p>
+                <img src="/img/Counter.svg">
+                <p>2015</p>
+              </div>
+              <p class="text-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+              <div class="technologys">
+                <ul class="list-technologys">
+                  <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+                  <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+                  <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
+                </ul>
+              </div>
+              <button type="button">See project</button>
+            </div>
+          </div>
+        </div>
+        </div>
+      </section>
+      <section class="section-p3">
+        <div class="about-me">
+          <div class="description1">
+            <h2>About Myself</h2>
+            <p>
+              I'm a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you like
+              what you see and have a project you need coded, don't hesitate to
+              contact me.
+            </p>
+          </div>
+          <div class="description2">
+            <p>lest's connect</p>
+            <ul class="social-container">
+              <li><a href="https://github.com/Kal2403"><img src="/img/github.svg" alt="" /></a></li>
+              <li><a href="https://www.linkedin.com/in/cristhian-castillo-41136322a/"><img src="/img/Linkedin.svg" alt="" /></a></li>
+              <li><a href="#"><img src="/img/thougn.svg" alt="" /></a></li>
+              <li><a href="https://twitter.com/Kal24031992"><img src="/img/Twitter.svg" alt="" /></a></li>
+              <li><a href="#"><img src="/img/M.svg" alt="" /></a></li>
+            </ul>
+            <button type="button">Get my resume</button>
+          </div>
+          <div class="languages">
+            <div class="name-drop">
+              <p>Lenguages</p>
```

Comment:
- [ ] kindly consider adding the  ("Languages", "Frameworks" and "Skills") in a ul and each in a li to follow HTML and CSS best practices and again for semantic reasons